### PR TITLE
Fix more links.

### PR DESCRIPTION
**Pull request by ScanuNicco**
This PR fixes wiki links in `readme.md`, `contribuiting.md`, and `issue_template.md` so that they point to this repo instead of the Adobe one. It also updates many of the GitHub links in Brackets itself to point at this repo instead of adobe/brackets. Finally, it also changes `app_name_about` (the name that shows up on the about dialog) to "Brackets Continued".

### CODE REVIEW

**Review by thornygravy — APPROVED** [review]
looks good

**Review by sprintr — APPROVED** [review]
LGTM

**Review by sprintr — APPROVED** [review]

**Review by thornygravy — APPROVED** [review]

**Review by thebiblelover7 — APPROVED** [review]
Seems good to me too

**Review by LehanMusthafa — APPROVED** [review]
Looks perfectly fine to me.

**Review by rileyhawk1417 — APPROVED** [review]
Looks nice and updated 👍🏾

**Comment by thebiblelover7** [comment]
@luke2m could you approve this PR. It needs an approving review "by reviewers with write access".

**Review by luke2m — APPROVED** [review]

**Comment by luke2m** [comment]
Looks good, but I don’t have write access. @yogo1212 ?

**Comment by thebiblelover7** [comment]
> Looks good, but I don’t have write access. @yogo1212 ?

oops, I thought you did....

**Comment by ScanuNicco** [comment]
I think it would be helpful if multiple people had write access to this repo. Maybe @luke2m or @sprintr?

**Comment by thebiblelover7** [comment]
> I think it would be helpful if multiple people had write access to this repo. Maybe @luke2m or @sprintr?

I agree

**Comment by yogo1212** [comment]
i also though being a member gave you write access..

File: src/brackets.config.json [hunk 1]
```diff
@@ -2,19 +2,19 @@
...
-        "glob_help_url"              : "https://github.com/adobe/brackets/wiki/Using-File-Filters",
-        "release_notes_url"          : "https://github.com/adobe/brackets/wiki/Release-Notes",
+        "how_to_use_url"             : "https://github.com/brackets-cont/brackets/wiki/How-to-Use-Brackets",
+        "support_url"                : "https://github.com/brackets-cont/brackets/wiki/Troubleshooting",
+        "suggest_feature_url"        : "https://github.com/brackets-cont/brackets/wiki/Suggest-a-Feature",
+        "get_involved_url"           : "https://github.com/brackets-cont/brackets/blob/master/CONTRIBUTING.md",
+        "glob_help_url"              : "https://github.com/brackets-cont/brackets/wiki/Using-File-Filters",
+        "release_notes_url"          : "https://github.com/brackets-cont/brackets/wiki/Release-Notes",
         "homepage_url"               : "http://brackets.io",
         "twitter_url"                : "https://twitter.com/brackets",
```
Comment (willnode) [inline src/brackets.config.json:14]: Pretty minor but I heard brackets people won't give us access to this Twitter account so somebody made https://twitter.com/BracketsCont

**Pull request merged**